server/pipeline/create.go

```diff
@@ -88,7 +88,7 @@ func Create(ctx context.Context, _store store.Store, repo *model.Repo, pipeline
 		return nil, ErrFiltered
 	} else if configFetchErr != nil {
 		log.Error().Str("repo", repo.FullName).Err(configFetchErr).Msgf("error while fetching config '%s' in '%s' with user: '%s'", repo.Config, pipeline.Ref, repoUser.Login)
-		return nil, updatePipelineWithErr(ctx, _forge, _store, pipeline, repo, repoUser, fmt.Errorf("could not load config from forge: %s", err.Error()))
+		return nil, updatePipelineWithErr(ctx, _forge, _store, pipeline, repo, repoUser, fmt.Errorf("could not load config from forge (%w): %s", err, err.Error()))
```

Comment:
Please add a test for it

Reply:
a unit test would not catch those issues.

full integration test suite will, but i paused my work on that :/

PS: yes you probably could write an unit test but that's just for this codeblock and it can just be done wront at all other places at another time -> thats why reviews are here :D

i dont think it is an hard issue and i did not test if we use some errors.Is or errors.As up in the caller stack, but its more a contract that somebody might at some point do so - e.g. we should expect golangs error handling system to work